apps/mobile/src/components/TradeChecklistFlow/components/CalculateAmountFlow/components/SetYourOwnPriceScreen/components/PriceInfo.tsx

```diff
@@ -14,20 +14,20 @@ interface Props {
 function PriceInfo({fiatTempValueAtom}: Props): JSX.Element | null {
   const {t} = useTranslation()
   const locale = getCurrentLocale()
-  const tradeBtcPrice = useAtomValue(tradeBtcPriceAtom)
   const fiatTempValue = Number(useAtomValue(fiatTempValueAtom)) ?? 0
+  const currentBtcPrice = useAtomValue(currentBtcPriceAtom)
 
-  const percentageDifference = useMemo(
-    () => 100 - (fiatTempValue / tradeBtcPrice) * 100,
-    [fiatTempValue, tradeBtcPrice]
+  const percentageDifference = calculatePercentageDifference(
+    fiatTempValue,
+    currentBtcPrice ?? 0
   )
 
   return fiatTempValue && fiatTempValue > 0 && percentageDifference !== 0 ? (
     <Info
       variant={'yellow'}
       hideCloseButton
       text={t(
-        percentageDifference < 0
+        percentageDifference > 0
```

Comment:
Check other conditions and unify when percentage difference is 0 to consistently return 'is higher/lower' text